rules/linux/lnx_security_tools_disabling.yml

```diff
@@ -16,6 +17,8 @@ detection:
           - 'chkconfig off iptables'
           - 'service ip6tables stop'
           - 'chkconfig off ip6tables'
+          - 'systemctl stop firewalld'
+          - 'systemctl disable firewalld'
         - CarbonBlack|contains:
           - 'service cbdaemon stop'
           - 'chkconfig off cbdaemon'
```

Comment:
I believe it make sense to split CommandLine and ProcessName in here, that would optimize the computing resources for the search. Also, as far as I know, there is a syslog message about deamon stopping. Could you please check this out?

Reply:
Made the split between the command line and the process name. Also I did check the syslog messages for stopping a daemon (systemd unit) and couldnt see an specific pattern in the syslog message as it might be taken some custom description from the daemon. See for example the messages stopping contianerd or osqueryd:

```
systemd[1]: Stopping containerd container runtime...
systemd[1]: Stopped containerd container runtime.
systemd[1]: Stopping The osquery Daemon...
systemd[1]: Stopped The osquery Daemon.
```

Do you think moving detections to use the syslog service might be a better approach in here?

Reply:
> Do you think moving detections to use the syslog service might be a better approach in here?

Well, it depends. I think that having both in place is a good option.

Reply:
Great idea, I just made it a global rule to include the syslog detection. Have not tried all those strings so might be also a good idea to rely on the community to fine tune the keywords. Let me know what you think :)

Reply:
Thank you very much, @alejandroortuno! (:

Reply:
I think that the rule is ready (: Well done!